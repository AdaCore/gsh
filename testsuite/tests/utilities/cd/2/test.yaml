title: Test for cd utility - nominal cases
