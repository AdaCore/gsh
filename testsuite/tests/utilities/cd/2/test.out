simple change dir: cd dir_a
currently in /private<TEST_DIR>/dir_a - path changed (true): 1

cd .
currently in /private<TEST_DIR> - path changed (false): 0

cd ./dir_a
currently in /private<TEST_DIR>/dir_a - path changed (true): 1

cd -
/private<TEST_DIR>/dir_a
currently in /private<TEST_DIR>/dir_a - path changed (true): 1

cd with no args
currently in /Users/vreboul - path changed (true): 1

cd with arg '..'
currently in /private<TEST_DIR>/dir_b - path changed (true): 1
